source/playbooks/get-started-with-playbooks.rst

```diff
@@ -0,0 +1,36 @@
+Get started with Mattermost Playbooks
+=====================================
+
+Access Mattermost playbooks from the Product Menu and select **Playbooks**.
+
+Playbooks are workflows that can be repeated and refined over time. They can be applied to a range of situations and give stakeholders visibility into progress, process, and metrics. Playbooks are configurable and editable. You can add tasks to checklists, include automation behaviors, and provide insights into your processes in the form of metrics.
+
+You can also choose a blank template and start from scratch - this is a good option if your use case is unique.
+
+Lastly, if you're unsure which option to choose, take a look at the **Learn how to use playbooks** template. This template breaks down the components of a playbook and you can also start a test run to see how everything fits together. If you're taking this option, you can stop reading here and enjoy the test run.
```

Comment:
I wonder if formatting this as a tip may be valuable to new Playbooks users?

Tip: Take a look at the Learn to use playbooks template. This template breaks down the components...